wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WDefinitionList.java

```diff
@@ -16,6 +16,8 @@
 public class WDefinitionList extends AbstractNamingContextContainer implements AjaxTarget,
 		SubordinateTarget, Marginable {
 
+	private static final String TERM_ATTRIBUTE = "WDefinitionList.term";
```

Comment:
Even though the change now sets the `term` on the `tag` and the `attribute`, I think planning ahead for `getTag()` being removed, it would still be good for the `TERM_ATTRIBUTE` to be made public and explain how they will need to access the `term` via the attribute in the change log.

It would also be good to include how to use the attribute in the javadoc of `WDefinitionList`.

Reply:
I don't understand why `TERM_ATTRIBUTE` should be made public. It should be considered an implementation detail. To add a term you can use `addTerm(String, WComponent...)` to get all of the terms you can use `getTerms()` or to get a specific term use `getComponentsForTerm(String)`.

Reply:
I could envision a completely different internal implementation where `WContainer`s were used as buckets for the components of each term, and each `WComponent` bucket, which is unknown to the user, has an attribute with the name of the term. In fact this might be a better implementation.

Reply:
You are right. The whole implementation of `WDefinitionList` smells as it originally hijacked the `getTag` method from the template logic and using the attribute is just as smelly. It might not even be worth doing this change until we consider how it could be refactored and deleting `getTag` is way off in the future anyway. Let me know what you think.